site/content/docs/sandbox/custom_sandbox/custom_environment/_index.md

```
@@ -103,4 +290,265 @@ response = sandbox.commands.run(payload)
+	}
+	return &Sandbox{ID: result.ID, client: c}, nil
+}
+
+// --- Sandbox ---
+
+type Sandbox struct {
+	ID     string
+	client *SandboxClient
+}
+
+func (s *Sandbox) Commands() *CommandService { return &CommandService{sandbox: s} }
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)